src/PrimeFuncPack.Core/core_objects/Objects.Tests/StringPredicateExtensionsTests/StringPredicateExtensionsTests.OrWhiteSpace.cs

```diff
@@ -0,0 +1,48 @@
+﻿#nullable enable
+
+using NUnit.Framework;
+using System;
+
+namespace PrimeFuncPack.Core.Objects.Tests
+{
+    partial class StringPredicateExtensionsTests
+    {
+        [Test]
+        public void IsNullOrWhiteSpace_SourceIsNull_ExpectTrue()
+        {
+            string? source = null;
+
+            var actual = source.IsNullOrWhiteSpace();
+            Assert.True(actual);
+        }
+
+        [Test]
+        public void IsNullOrWhiteSpace_SourceIsEmpty_ExpectTrue()
+        {
+            string source = string.Empty;
+
+            var actual = source.IsNullOrWhiteSpace();
+            Assert.True(actual);
+        }
+
+        [Test]
+        [TestCase(" ")]
+        [TestCase("   ")]
+        public void IsNullOrWhiteSpace_SourceIsWhiteSpace_ExpectTrue(
```

Comment:
Shall we add some whitespace test cases different from just SPACE - like TAB ("\t")?

Reply:
Have done